### PR TITLE
Reduce libs

### PR DESCRIPTION
**Pull request by Beagle**
265 lines of code in 3 lib_*.php files remain after this merge

### CODE REVIEW

File: deploy/lib/data/Message.php [hunk 1]
```diff
@@ -159,4 +159,25 @@ public static function sendChat($user_id, $msg) {
 
         return false;
     }
+
+    /**
+     * @return int Number of rows deleted
+     */
+    public static function deleteOldMessages() {
+        $statement = query("delete from messages where date < ( now() - '3 months'::interval)");
```
Comment (tchalvak) [inline deploy/lib/data/Message.php]: Hmm, going to open an issue to stop deleting old messages.

File: deploy/lib/environment/lib_assert.php [hunk 2]
```diff
@@ -1,13 +0,0 @@
-<?php
-/**
- * Assert lib file to suppress assert warnings on live servers.
- * 
- * @package lib
- * @subpackage settings
-**/
-
-if (!DEBUG) {
-	assert_options(ASSERT_ACTIVE, 0); // *** Sets assert off on live.
```
Comment (tchalvak) [inline deploy/lib/environment/lib_assert.php]: Isn't this needed?

Comment (tchalvak) [inline deploy/lib/environment/lib_assert.php]: Ah, I see, it was moved to the bootstrap.php file.

File: deploy/lib/data/lib_db.php [hunk 3]
```diff
@@ -40,63 +40,33 @@ function query($sql, $bindings=array(), $return_resultset=true) {
...
+/**
+ * Wrapper to get a multi-dimensional array.
+ */
 function query_array($sql_query, $bindings=array()) {
-	return query($sql_query, $bindings, false); // Set return_resultset to false to return the array.
+	return query($sql_query, $bindings)->fetchAll(PDO::FETCH_ASSOC);
 }
 
-// Insert sql, returns the id insert by default.
-function insert_query($insert_query, $bindings=array(), $sequence_name){
```
Comment (tchalvak) [inline deploy/lib/data/lib_db.php]: Why did you remove this?

**Comment by tchalvak** [comment]
@beagle Tests fail.

File: deploy/lib/data/lib_db.php (hunk 3 above)
Comment (Beagle) [inline deploy/lib/data/lib_db.php]: It was only being used in one place and that one place wasn't being used
itself. Every other insert query uses a different method.
